tests/e2e/e2e.go

```diff
@@ -404,6 +408,42 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 	framework.ExpectNoError(err, "failed to wait for tidb-operator to be uninstalled")
 })
 
+// TODO: refactor it to combine with code in /tests/e2e/br/framework/framework.go
+func ForceCleanBackups(kubeClient kubernetes.Interface, extClient versioned.Interface, apiExtClient apiextensionsclientset.Interface) error {
+	if _, err := apiExtClient.ApiextensionsV1().CustomResourceDefinitions().Get("backups.pingcap.com", metav1.GetOptions{}); err != nil {
+		if !errors.IsNotFound(err) {
+			return err
+		}
+		return nil
+	}
+	nsList, err := kubeClient.CoreV1().Namespaces().List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, item := range nsList.Items {
+		ns := item.Name
+		bl, err := extClient.PingcapV1alpha1().Backups(ns).List(metav1.ListOptions{})
+		if err != nil {
+			framework.Logf("failed to list backups in namespace %s: %v", ns, err)
+			return err
+		}
+		for i := range bl.Items {
+			name := bl.Items[i].Name
+			if err := extClient.PingcapV1alpha1().Backups(ns).Delete(name, nil); err != nil {
+				framework.Logf("failed to delete backup(%s) in namespace %s: %v", name, ns, err)
+				return err
+			}
+			// use patch to avoid update conflicts
+			patch := []byte(`[{"op":"remove","path":"/metadata/finalizers"}]`)
+			if _, err := extClient.PingcapV1alpha1().Backups(ns).Patch(name, types.JSONPatchType, patch); err != nil {
```

Comment:
It seems that `PropagationPolicy` can only affect GC with `ownerReferences`

Reply:
can you add a comment to elaborate on why we need this patch(what the set in  finalizers?), since generally we should not do directly remove the finalizers.

Reply:
If we delete namespace directly, all resource will be deleted in this namespace. However, backup finalizer is depend on some resource such as `Secret` in the namespace, so it will failed and block namespace deletion.

Reply:
add comment in the code?